### PR TITLE
New PR - Audit & fix Information.md

### PR DESCRIPTION
**Pull request by Yoda-Canada**
I closed the PR #81 and open the new PR for Audit & fix Information.md page #44

Steps done to modify and check Information.md:

1. Fixed images work in dark modes. The memory model image's downward arrows and the letter in the bottom right corner are not clear in dark modes. I wrapping them in a <span> tag and adding class "mdImg" to it. 
2. The images can't display in all models. Change all the image path from "/img/.jpg" to "../../static/img/.jpg". The issue is fixed.
3. Fix Frontmatter for the page to include id, title, slug, and description.
4. Replace blockquote with Admonitions for clarity.
5. Added alt text for diagram images.
6. Some of " * "will get converted into _ after being formatted with Prettier.  I use \* replace them.

### CODE REVIEW

File: docs/A-Introduction/information.md [hunk 1]
```diff
@@ -78,33 +87,44 @@ To learn how to convert between hexadecimal and binary refer to the chapter enti
...
 ### Addresses
 
 Each byte of RAM has a unique address.  Addressing starts at zero, is sequential, and ends at the address equal to the size of RAM less 1 unit. 
 
 For example, 4 Gigabytes of RAM
 
-* consists of 32 (= 4 * 8) Gigabits
+* consists of 32 (= 4 \* 8) Gigabits
 * starts at a low address of 0x00000000
 * ends at a high address of 0xFFFFFFFF
```
Comment (hlavu) [inline docs/A-Introduction/information.md:104]: I think address values in these lines should be wrapped in backticks.

File: docs/A-Introduction/information.md [hunk 2]
```diff
@@ -78,33 +87,44 @@ To learn how to convert between hexadecimal and binary refer to the chapter enti
...
-* Tera or T (=1024G): 1 Terabyte = 1024 * 1024 * 1024 * 1024 bytes ~ 10<sup>12</sup> bytes
-* Peta or P (=1024T): 1 Petabyte = 1024 * 1024 * 1024 * 1024 * 1024 bytes ~ 10<sup>15</sup> bytes
-* Exa or E (=1024P): 1 Exabyte = 1024 * 1024 * 1024 * 1024 * 1024 * 1024 bytes ~ 10<sup>18</sup> bytes
+* Mega or M (=1024k): 1 Megabyte = 1024 \* 1024 bytes ~ 10<sup>6</sup> bytes
+* Giga or G (=1024M): 1 Gigabyte = 1024 \* 1024 \* 1024 bytes ~ 10<sup>9</sup> bytes
+* Tera or T (=1024G): 1 Terabyte = 1024 \* 1024 \* 1024 \* 1024 bytes ~ 10<sup>12</sup> bytes
+* Peta or P (=1024T): 1 Petabyte = 1024 \* 1024 \* 1024 \* 1024 \* 1024 bytes ~ 10<sup>15</sup> bytes
+* Exa or E (=1024P): 1 Exabyte = 1024 \* 1024 \* 1024 \* 1024 \* 1024 \* 1024 bytes ~ 10<sup>18</sup> bytes
 
 Note that the multiplying factor is 1024, not 1000.  1024 bytes is 2<sup>10</sup> bytes, which is approximately 10<sup>3</sup> bytes.
```
Comment (lmpham1) [inline docs/A-Introduction/information.md:129]: I think this line can be wrapped in admonition as well

File: docs/A-Introduction/information.md [hunk 3]
```diff
@@ -78,33 +87,44 @@ To learn how to convert between hexadecimal and binary refer to the chapter enti
 
 The memory model for organizing information stored in RAM is linear.  Any byte in memory is accessible through a map that treats each actual physical memory location as a position in a continuous sequence of locations aligned next to one another.
 
+<span className="mdImg" align="center" alt="A memory model image">
+
 ![RAM memory model: linearly mapped \(continuous sequence\)](https://ict.senecacollege.ca//~ipc144/pages/images/ram.png)
```
Comment (humphd) [inline docs/A-Introduction/information.md:92]: This image needs to be brought into the repo

**Review by humphd — CHANGES_REQUESTED** [review]

File: docs/A-Introduction/information.md (hunk 3 above)
Comment (Yoda-Canada) [reply docs/A-Introduction/information.md:92]: I have modified it, thx.

File: docs/A-Introduction/information.md (hunk 1 above)
Comment (Yoda-Canada) [reply docs/A-Introduction/information.md:104]: I have modified it, thx.

File: docs/A-Introduction/information.md (hunk 3 above)
Comment (humphd) [reply docs/A-Introduction/information.md:92]: I don't understand, it's still pointing to https://ict.senecacollege.ca//~ipc144/pages/images/ram.png.  All images need to get imported into this repo and added to git.

File: docs/A-Introduction/information.md (hunk 1 above)
Comment (humphd) [reply docs/A-Introduction/information.md:104]: You haven't.  Please make the correction.

File: docs/A-Introduction/information.md (hunk 2 above)
Comment (humphd) [reply docs/A-Introduction/information.md:129]: This needs to get done still.

**Review by humphd — CHANGES_REQUESTED** [review]
Please don't say you have done things if they are not done.  Let me know when these changes have been pushed.

File: docs/A-Introduction/information.md (hunk 1 above)
Comment (Yoda-Canada) [reply docs/A-Introduction/information.md]: I have modified, thx

**Comment by Yoda-Canada** [comment]
@humphd I see. Next time, I will upload all of them successfully and then reply.  I still confuse the rebase command, and still can't merge all commits.
1. Yesterday, I create a new branch named issue-44 and modified all contents in this branch. I rebase all commits and uploaded with one commit successfully.
2. Today, I find there are new reviews, then I modified them and rebase all commits to one successfully.
3. I use the command "git push origin issue-44". The upload failed because lots of new commits ahead of this commit.
4. I use " git checkout main" -> "git pull upstream main" -> "git merge issue-44" -> "git push origin main". It uploaded successfully. The PR #101 can't Synchronization Update because it PR from issue-44 branch.
5. Then I "git pull origin issue-44". There are a lot of new commits from upstream and they mixed with my commit. I searched some methods from the website, but it still can't be one. 
Now all reviews are modified and updated, but 6 commits can't be one. I have no idea how to deal with it. Maybe it would be better to create a new one.

**Comment by TueNguyen2911** [comment]
@Yoda-Canada. 
4. You should have merged the `main` branch into `issue-44` and resolved any conflicts and then pushed it to automatically update your PR
5. 
- To fix this, I think you can try is going to your `issue-14`, use `git reflog`, check [here](https://medium.com/@shreyawhiz/how-to-undo-a-mistaken-git-rebase-life-saver-2977ff0a0602) for more information. Revert your squashed commit by using `git reset — hard <commit sha hash>`, make changes, `rebase` and push it to issue-44 again. Before doing this, you can checkout a new branch to be extra safe
- If it doesn't fix, you can try cloning your project again and repeat the instruction above

**Comment by Yoda-Canada** [comment]
> @Yoda-Canada. 4. You should have merged the `main` branch into `issue-44` and resolved any conflicts and then pushed it to automatically update your PR 5.
> 
> * To fix this, I think you can try is going to your `issue-14`, use `git reflog`, check [here](https://medium.com/@shreyawhiz/how-to-undo-a-mistaken-git-rebase-life-saver-2977ff0a0602) for more information. Revert your squashed commit by using `git reset — hard <commit sha hash>`, make changes, `rebase` and push it to issue-44 again. Before doing this, you can checkout a new branch to be extra safe
> * If it doesn't fix, you can try cloning your project again and repeat the instruction above

@TueNguyen2911 Thank you so much, but it can't resolve the issue.

**Comment by TueNguyen2911** [comment]
@Yoda-Canada, I'll add yours as `remote` to my local and try to resolve this issue, I'll get back to you if I succeed

**Comment by Yoda-Canada** [comment]
> 

@TueNguyen2911 oh, I have already delete the rpo

**Comment by humphd** [comment]
@Yoda-Canada have you opened a third PR for this?  Please take some time to figure out what you're doing wrong with git so this isn't necessary.  You should never need to delete a repo.

**Comment by humphd** [comment]
Closing in favour of #105.  I think all the comments here are addressed there.

**Pull request closed without merging**